code/datums/components/crafting/equipment.dm

```diff
@@ -57,12 +57,29 @@
 	time = 20 SECONDS
 	category = CAT_EQUIPMENT
 
-/datum/crafting_recipe/freezer_cabinat
-	name = "Freezer Cabinet"
+/datum/crafting_recipe/secured_freezer_cabinet
+	name = "Secured Freezer Cabinet"
```

Comment:
```suggestion
	name = "Secure Freezer Cabinet"
```
secured = bolted down or otherwise safe in place
secure = ideal for safekeeping, also consistent with the current secure lockers